libs/providers/multi-provider/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@openfeature/multi-provider",
```

Comment:
I wonder if we should add `server` in this name to make it clear that it won't work on the web.

Reply:
I was kind of following the naming conventions I saw in the rest of the repo, where the "server" version of things didn't specify server, but the "web" version did specify "web". I assume that's because most of the server stuff was created before "web" existed. Should I go with consistency with the others, or just name it with "server" for clarity?